webpack.common.js

```diff
@@ -94,14 +95,15 @@ module.exports = (cssFilename, outputFilename, minimizeCss) => ({
           MiniCssExtractPlugin.loader,
           {
             loader: 'css-loader',
-            options: {
-              minimize: minimizeCss,
-            },
           },
           {
             loader: 'postcss-loader',
             options: {
-              plugins: [pxtorem({ propList: ['*'] }), autoprefixer()],
+              plugins: [
+                pxtorem({ propList: ['*'] }),
+                autoprefixer(),
+                minimizeCss ? cssnano : false,
```

Comment:
What does a value of `false` do, does it cause webpack to just ignore the array item?